### ISSUE TITLE
InsertExpression.defaultValues is unexpectedly never

### ISSUE THREAD

**Issue by lorefnon** [juanluispaz/ts-sql-query#40]
For a table defined as follows: 

```ts
class TestTable extends Table<DBConnection, "Test"> {
    id = this.autogeneratedPrimaryKey("id", "int")
    foo = this.optionalColumnWithDefaultValue("foo", "string")

    constructor() { super("test")}
}
```

I'd be expect to be able to insert with default values because there are no mandatory columns.

However, attempting to write something like: 

```ts
getConnection().insertInto(new TestTable()).defaultValues()
```
reveals that defaultValues is `never` and hence not callable. 

More interestingly: 

```
type T1 = RequiredColumnsForSetOf<TestTable> // never
type T2 = keyof RequiredColumnsForSetOf<TestTable> // string | number | symbol
```

Hence [this condition](https://github.com/juanluispaz/ts-sql-query/blob/a8f67bf4ad71a9ea2b4b62edeee21a72aef6d920/src/expressions/insert.ts#L501) never holds true. 

I am not sure but could it be that `keyof never` has changed across ts versions ?

**Comment by lorefnon** [comment]
So this did change in TS, but the change was intended. Hence we'll need to update the implementation here.

But it also looks like the keyof was unnecessary to begin with. RequiredColumnsForSetOf returns a union of keys or never, so we can skip keyof.

**Comment by juanluispaz** [comment]
Released in ts-sql-query 1.29.0
